src/assets/config.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
+
+const ML_BEST_OF = 'ml_best_of';
+const ML_EVENTS = 'ml_events';
+const ML_DONORS = 'ml_donors';
+
+export default {
+  ML_BEST_OF: {
```

Comment:
suggestion: can we make an interface for this property value? that way we can easily share type

Reply:
yes, I added an interface type in  models.ts